core/src/backend/navigator.rs

```diff
@@ -42,14 +42,14 @@ pub enum SocketMode {
 /// The handling mode of links opening a new website.
 #[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
 #[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
-pub enum OpenURLMode {
```

Comment:
Also, an `OpenLinks` enum feels weird. I think this should be kept as `OpenURLMode` (or at least `OpenLinkMode`).